RecoTracker/IterativeTracking/python/LowPtTripletStep_cff.py

```diff
@@ -95,6 +95,21 @@
     CAPhiCut = 0.05,
 ))
 
+trackingPhase2PU140.toModify(lowPtTripletStepHitDoublets, layerPairs = [0,1]) # layer pairs (0,1), (1,2)
+trackingPhase2PU140.toReplaceWith(lowPtTripletStepHitTriplets, _caHitTripletEDProducer.clone(
```

Comment:
this could be grouped with ``` for mod in {trackingPhase1, trackingPhase2PU140}: mod.toModify ... ```
Well, eventually, I guess there is a reason to fine-tune individually. So, OK.